label_studio/data_export/mixins.py

```diff
@@ -319,11 +319,11 @@ def convert_file(self, to_format):
                 output_file = pathlib.Path(tmp_dir) / (str(out_dir.stem) + '.zip')
                 filename = pathlib.Path(input_name).stem + '.zip'
 
-            out = read_bytes_stream(output_file)
-            return File(
-                out,
-                name=filename,
-            )
+            with open(output_file, mode='rb') as f:
+                return File(
+                    io.BytesIO(f.read()),
```

Comment:
is it ok to use f.read() here? it will load the whole file into the memory again..

Reply:
I basically just unrolled what `read_bytes_stream` used to do here. I'm not sure exactly how to fix this part right now, but I don't think it's relevant to the issue I was trying to resolve in this PR. Added a comment suggesting taking a look in future.